bin/main.rb

```diff
@@ -1,5 +1,13 @@
-# rubocop:disable Layout/LeadingCommentSpace, Lint/MissingCopEnableDirective, Metrics/MethodLength, Style/GlobalVars, Style/ClassVars, Style/MultipleComparison, Naming/VariableNumber, Layout/LineLength
-#!/usr/bin/env ruby
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Style/GlobalVars
+# rubocop:disable Style/ClassVars
+# rubocop:disable Style/MultipleComparison
+# rubocop:disable Naming/VariableNumber
+# rubocop:disable Layout/LineLength
+# !/usr/bin/env ruby
+
+require_relative '../lib/logic_game.rb'
+
 class Board
```

Comment:
- [ ] All of the game-logic resources (class/modules) should be in the lib directory and not in the bin directory.
screenshot of the requirements 👇 
![Screen Shot 2020-10-01 at 11 05 47 PM](https://user-images.githubusercontent.com/48804258/94863374-a652f180-043a-11eb-9db2-24d2bb5d0392.png)

Reply:
I was not able to play your game. When I ran bin/main.rb I got the following errors point_down
the problem here is because the shebang only works on Windows systems if I am wrong please help me to have the right answer